Library/Formula/gnu-units.rb

```diff
@@ -28,3 +32,37 @@ def install
     assert_equal "* 18288", shell_output("#{bin}/gunits '600 feet' 'cm' -1").strip
   end
 end
+
+# Temporary fix for "invalid/nonprinting UTF-8" warnings on startup,
+# see #20297.
+#
+# The current maintainer of GNU units is aware of the issue and has
+# fixed it in the (as of 2015-04-08) unreleased development version
+# which will be released later this year.
+#
+# See #38454 for details.
```

Comment:
Coud you put these comments above the `patch :DATA` above and provide a full link to the issue? Can you also note when you submitted the bug report and the name of the person? Once we have that we should be good to go.